android/build.gradle

```diff
@@ -48,6 +48,7 @@ android {
 repositories {
   mavenCentral()
   jcenter()
+  google()
```

Comment:
@Salakar Do we need this ?

Reply:
Nope, there's only 1 dependency, and it's `org.jetbrains.kotlin` which comes from Maven central: https://mvnrepository.com/artifact/org.jetbrains.kotlin

Reply:
I added this line because it was suggested by Android Studio to resolve a Gradle sync error when first syncing the project from master. I can try to replicate and provide steps.

Reply:
This is what I get when cloning master and opening the `android` directory in Android Studio:

<img width="838" alt="screen shot 2019-01-17 at 12 13 34 am" src="https://user-images.githubusercontent.com/1173161/51299097-cf281e80-19ec-11e9-9ea8-705f24ab6d17.png">

<img width="427" alt="screen shot 2019-01-17 at 12 12 35 am" src="https://user-images.githubusercontent.com/1173161/51299103-d51dff80-19ec-11e9-9fcb-2c4c36d03f43.png">